cms/src/components/images/ImageBrowser.vue

```diff
@@ -0,0 +1,71 @@
+<script setup lang="ts">
+import { db } from "@/db/baseDatabase";
+import { DocType, type ImageDto, type Uuid } from "@/types";
+import LImage from "./LImage.vue";
+import LButton from "../button/LButton.vue";
+import { PlusIcon } from "@heroicons/vue/20/solid";
+import LCard from "../common/LCard.vue";
+import ImageEditor from "./ImageEditor.vue";
+import { ref } from "vue";
+
+// TODO: Implement select event
+
+const baseUrl: string = import.meta.env.VITE_CLIENT_IMAGES_URL;
+// TODO: Replace with a generic fallback image included in project source code
+const fallbackImg: string =
+    "https://cdn.activechristianity.org/image/upload/c_thumb,w_600,g_center/v1666007697/en_ke/f7bcwvet6axzmih97pv8.jpg";
```

Comment:
Would be nice to solve this todo before we merge this 😄

Reply:
Yes, agree